piksi_tools/serial_link.py

```diff
@@ -107,6 +110,20 @@ def get_logger(use_log=False, filename=LOG_FILENAME):
   print "Logging at %s." % filename
   return JSONLogger(filename)
 
+def get_append_logger(filename):
```

Comment:
Have you looked into what manner of appender might work here? I've only looked at https://docs.python.org/2/library/logging.handlers.html#rotatingfilehandler and https://docs.python.org/2/library/logging.handlers.html#timedrotatingfilehandler, but don't know if there's a better option tied to fluentd.